core/ecschema-metadata/src/test/Deserialization/XmlParser.test.ts

```diff
@@ -1868,6 +1868,61 @@ describe("XmlParser", () => {
     });
 
     describe("Property Parsing Tests", () => {
+      describe("String Parsing Tests", () => {
+        it("With value set to valid string, CustomAttributeProvider should provide valid instance.", async () => {
+          const itemXml = `
+            <ECCustomAttributes>
+              <TestAttribute xmlns="TestSchema.1.0">
```

Comment:
Is this an older schema? I think we require 3 digits in EC 3.2 schemas, and 2 digits in EC 3.1 and earlier

Reply:
All the XmlParser tests have this.  Should I update all of them?

Reply:
xmlns should include all three digits of the schema version.  At least update this test so we don't make the world worse even if we're not making it better.

Reply:
I updated all of them.